src/main/java/ui/components/pickers/MilestonePickerDialog.java

```diff
@@ -210,8 +210,10 @@ private void updateNewlyAssignedMilestone(Optional<PickerMilestone> selectedMile
         HBox newlyAssignedMilestoneBox = createNewlyAssignedMilestoneBox();
         assignedMilestoneStatus.getChildren().add(newlyAssignedMilestoneBox);
 
-        if (!selectedMilestone.isPresent()) return;
-
+        if (!selectedMilestone.isPresent()) {
+            assignedMilestoneBox.setStyle("-fx-border-radius: 3;-fx-border-style: dotted;-fx-alignment:center");
```

Comment:
Hmm, maybe these style strings could be refactored into `private static` `NO_BORDER_STYLE` and `DOTTED_BORDER_STYLE`?